mne/io/utils.py

```diff
@@ -305,6 +305,6 @@ def _construct_bids_filename(base, ext, part_idx):
             idx = deconstructed_base.index(mod)
             modality = deconstructed_base.pop(idx)
     base = '_'.join(deconstructed_base)
-    use_fname = '%s_part-%02d_%s%s' % (base, part_idx, modality, ext)
+    use_fname = '{}_split-{:02}_{}{}'.format(base, part_idx, modality, ext)
```

Comment:
We can use f-strings now :) I think it makes this line much easier to read, but I'll leave this up to you – your PR ;)
```suggestion
    use_fname = f'{base}_split-{part_idx:02}_{modality}{ext}'
```

Reply:
:heart_eyes:

Reply:
Only do this if we don't plan on backporting

Reply:
uff, actually backporting would be nice ... damnit. I'll revert the last commit.

Reply:
Oh noes. So sorry

Reply:
@larsoner to which MNE versions can we backport this? and how much effort is it? ideally we'd backport it to the version that first introduced `split_naming='bids'` and all following ones :thinking: 

related: do we have stats (histogram) which MNE versions are mostly used (downloaded)?

Reply:
> do we have stats (histogram) which MNE versions are mostly used (downloaded)?

No idea...